shoop/core/models/_orders.py

```diff
@@ -279,6 +281,19 @@ def __str__(self):  # pragma: no cover
         else:
             return "Order %s (%s)" % (self.identifier, name)
 
+    @property
+    def codes(self):
```

Comment:
if the field is `codes` this could be function instead of property for example `get_codes`